File: output/CodeLlama/codenet/C++/Python/s794872785.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import sys
import math
import string


def solve():
    maxsize = 0
    maxcnt = 0
    ans_num = 0
    ans_num2 = 0
    str_v = []

    while True:
        in = sys.stdin.readline().strip()
        if in == '':
            break
        str_v.append(in)
        if maxsize < len(in):
            maxsize = len(in)
            ans_num = len(str_v) - 1

    for i in range(len(str_v)):
        cnt = 0
        for j in range(len(str_v)):
            if str_v[i] == str_v[j]:
                cnt += 1
        if maxcnt < cnt:
            maxcnt = cnt
            ans_num2 = i

    print str_v[ans_num2], str_v[ans_num]


if __name__ == '__main__':
    solve()

File: output/CodeLlama/codenet/C++/Java/s794872785.java
import java.util.*;
import java.io.*;


public class s794872785 {

	public static void main(String[] args) {
		Scanner cin = new Scanner(System.in);
		int maxsize=0;
		int maxcnt=0;
		int ans_num;
		int ans_num2;
		ArrayList<String> str_v = new ArrayList<String>();
		
		String in;
		while(cin.hasNext()){
			in = cin.next();
			str_v.add(in);
			if(maxsize < in.length()){
				maxsize = in.length();
				ans_num = str_v.size()-1;
			}
		}
		for(int i=0;i<str_v.size();i++){
			int cnt=0;
			for(int j=0;j<str_v.size();j++){
				if(str_v.get(i).equals(str_v.get(j))) cnt++;
			}
			if(maxcnt < cnt){
				maxcnt = cnt;
				ans_num2 = i;
			}
		}
		
		System.out.println(str_v.get(ans_num2) + " " + str_v.get(ans_num));
		
		cin.close();
	}

}